cvat-sdk/.gitignore

```diff
@@ -74,4 +74,4 @@ cvat_sdk/api_client/
 requirements/
 docs/
 setup.py
-README.md
\ No newline at end of file
+README.md
```

Comment:
Why did you put setup.py and README.md into .gitignore?

Reply:
They are also generated automatically. We decided to remove such files. Moreover, `setup.py` is meaningless without REST client code.